api/prometheus/v1/api_bench_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
Not that it matters a lot, but for new files, we usually put the current year here.